cmd/engine/main.go

```diff
@@ -783,7 +783,7 @@ func newController(ctx context.Context, c *cli.Context, cfg *config.Config, pubs
 	}
 
 	bkLogsW := io.Discard
-	if cfg.Debug {
+	if c.IsSet("extra-debug") {
```

Comment:
We could also do `if slog.Default().Enabled(ctx, slog.LevelExtraDebug)` - this means it gets automatically enabled if `trace` is turned on.

Reply:
nice, will do that instead